testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/transaction/usertransaction/UserTransactionAccessTestCase.java

```diff
@@ -46,7 +47,7 @@
     public static JavaArchive createDeployment() {
         final JavaArchive jar = ShrinkWrap.create(JavaArchive.class, MODULE_NAME + ".jar");
         jar.addPackage(UserTransactionAccessTestCase.class.getPackage());
-        jar.as(ZipExporter.class).exportTo(new File(".", jar.getName()), true);
+        jar.as(ZipExporter.class).exportTo(new File(TestSuiteEnvironment.getTmpDir(), jar.getName()), true);
```

Comment:
I think this like should have just been deleted, there is no reason to export the archive

Reply:
deleted.